packet/flatcar-linux/kubernetes/calico/host-endpoint-controller.yaml

```diff
@@ -27,7 +27,7 @@ spec:
     spec:
       serviceAccountName: calico-hostendpoint-controller
       containers:
-      - image: quay.io/kinvolk/calico-hostendpoint-controller:v0.0.2
+      - image: kinvolk/calico-hostendpoint-controller:v0.0.2
```

Comment:
Shouldn't that be 0.0.3 by now?

Reply:
Comes through the other PR that is open.